packages-ts/terra-sdk/test/index.test.ts

```diff
@@ -0,0 +1,67 @@
+        expect(got).toEqual([{
+            answer:null,
+            round:null,
+            epoch: null,
+            aggregatorRoundId:null,
+            observationsTS:null,
+        }]);
+    })
+    it('parseLog missing event', () => {
+        let got = OCR2Feed.parseLog('[{"events":[{"type":"execute_contract","attributes":[{"key":"sender","value":"terra167h3sh8c4pgs8grxz24pam2x764flydv3h9pd8"},{"key":"contract_address","value":"terra1fwhxcdlm7cefu585pv54vgcpacq9daxppa823c"}]},{"type":"from_contract","attributes":[{"key":"contract_address","value":"terra1fwhxcdlm7cefu585pv54vgcpacq9daxppa823c"},{"key":"method","value":"transmit"},{"key":"method","value":"transmit"}]},{"type":"message","attributes":[{"key":"action","value":"/terra.wasm.v1beta1.MsgExecuteContract"},{"key":"module","value":"wasm"},{"key":"sender","value":"terra167h3sh8c4pgs8grxz24pam2x764flydv3h9pd8"}]},{"type":"wasm","attributes":[{"key":"contract_address","value":"terra1fwhxcdlm7cefu585pv54vgcpacq9daxppa823c"},{"key":"method","value":"transmit"},{"key":"method","value":"transmit"}]},{"type":"wasm-transmitted","attributes":[{"key":"contract_address","value":"terra1fwhxcdlm7cefu585pv54vgcpacq9daxppa823c"},{"key":"config_digest","value":"000283aa7440f0e70a44e5e122a6a4112b4670c9a72f8746eeda1cc93c296b1a"}]}]}]');
+        expect(got).toEqual([]);
+    })
+    // requires a WS_URL for Columbus-5
```

Comment:
Is there a convention for skipping integration tests when the resource is not available? Or some way we can do this live in CI?

Reply:
We can just skip it for now with something like:

```js
    (process.env.WS_URL ? it : it.skip)('onRound', async () => {
```